core/scripts/liquidity-mining/CalculateBalancerLPProviders.js

```diff
@@ -26,7 +26,7 @@ const argv = require("minimist")(process.argv.slice(), {
 });
 
 const UMA_PER_WEEK = toBN(toWei("25000"));
-const BLOCKS_PER_SNAPSHOT = 64;
+const BLOCKS_PER_SNAPSHOT = 256;
```

Comment:
OOC how much speed up does this provide?

Reply:
run time on my machine went from about 40 mins to less than 10